pages/home/crewai/custom-auth-flow.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Custom Auth Flow with CrewAI"
+description: "Learn how to create a custom auth flow with CrewAI"
+---
+
+import { Steps } from "nextra/components";
+
+## Custom Auth Flow with CrewAI
+
+In this guide, we will explore how to create a custom auth flow that will be performed before executing Arcade tools within your CrewAI agent team. Follow the step-by-step instructions below.
```

Comment:
I'm missing the "why" here -- why should I build a custom auth flow? Does it solve a specific problem that CrewAI developers will run into? (How will I know if I need it?)

Reply:
added